### PR TITLE
Storestats

### PR DESCRIPTION
**Pull request by railsagainstignorance**
persisting the GAMES_STATS obj in the db

### CODE REVIEW

File: bin/lib/gameLonger.js [hunk 1]
```diff
@@ -83,10 +85,7 @@ class Game{
 		const missing_fields = [];
 
     // handle when we are re-building a Game instance from a simple obj (e.g. from the DB)
-		if( config === undefined ) {
-			GAMES_STATS.counts.created += 1;
-		} else {
-			GAMES_STATS.counts.cloned += 1;
+		if( config !== undefined ) {
 			if (userUUID !== config['uuid']) {
 				throw `Game.constructor: config defined, but mistmatched uuids: userUUID=${userUUID}, config.uuid=${config.uuid}, config=${JSON.stringify(config)}`;
```
Comment (seanmtracey) [inline bin/lib/gameLonger.js]: Is this ever caught anywhere? From what I can see, this isn't in a promise chain - throwing the error could bring down the server. Is that intentional?

File: bin/lib/gameLonger.js [hunk 2]
```diff
@@ -329,53 +328,85 @@ class Game{
...
+	static writeToDB( objWithUuid ) {
 		return new Promise( (resolve, reject) => {
-			const objType = typeof game;
-			if (! objType === 'Game') {
-				reject( `Game.writeToDB must be passed an obj of type Game, but it was type: ${objType}` );
+			if (! objWithUuid.hasOwnProperty('uuid')) {
+				reject( `Game.writeToDB must be passed an obj with a uuid field, objWithUuid=${JSON.stringify(objWithUuid)}` );
 			} else {
-				database.write(game, process.env.GAME_TABLE)
+				database.write(objWithUuid, process.env.GAME_TABLE)
```
Comment (seanmtracey) [inline bin/lib/gameLonger.js]:
```
database.write(objWithUuid, process.env.GAME_TABLE)
    .then( () => { resolve(); })
;
``` 
Nice formatting

File: bin/lib/gameLonger.js [hunk 3]
```diff
@@ -26,11 +26,13 @@ intervalDays - how many days of articles are covered by the current correlations
 history - record the sequence of questionData items for a summary at the end of a game
 achievedHighestScore, achievedHighestScoreFirst - set when finishing a question, based on current best score
 */
+const GAMES_STATS_ID = 'UUID_FOR_GAMES_STATS';
 
-const GAMES_STATS = {
+let GAMES_STATS = {
```
Comment (railsagainstignorance) [inline bin/lib/gameLonger.js]: is modified by updateGamesStats

File: bin/lib/gameLonger.js (hunk 1 above)
Comment (railsagainstignorance) [inline bin/lib/gameLonger.js]: the throw would only happen when constructor is cloning a game, and is now caught

**Pull request merged**